test/integration/tcp_proxy_integration_test.cc

```diff
@@ -94,8 +94,6 @@ TEST_P(TcpProxyIntegrationTest, TcpProxyLargeWrite) {
       test_server_->store()
           .counter("cluster.cluster_with_buffer_limits.upstream_flow_control_resumed_reading_total")
           .value();
-  EXPECT_LT(0, upstream_pauses);
```

Comment:
This deflakes the test.  On maybe 1:5000 runs (1:50 on tsan) the threading would work out such that the socket was drained fast enough it didn't back up, and so the varz were legitimately 0.